test/pytest/test_gRPC_inference_api.py

```diff
@@ -7,7 +7,7 @@
 import test_utils
 
 
-inference_data_json = "/../postman/inference_data.json"
+inference_data_json = "../postman/inference_data.json"
```

Comment:
Might as well fix this too?

Reply:
It works fine with the join it appears in later, as I tried with another example in python `os.join`